lib/presentation/bookPages/view/book_page_view.dart

```diff
@@ -28,10 +32,18 @@ class _BookPageView extends State<BookPageView> {
   List<BookPage> bookPagesList = [];
   List<BookChapter> bookChaptersList = [];
   GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+  bool isMessageReceived = false;
```

Comment:
Consider using getter instead of flag.

Reply:
I think we can't use getter here because we need it to  be set on false latter in DisplayBookstate so it. does not. trigger again when we swipe the page for example.

Reply:
I see. I get it.